logger/build.gradle

```diff
@@ -49,4 +49,5 @@ dependencies {
   testCompile 'junit:junit:4.12'
   testCompile "org.robolectric:robolectric:3.0"
   testCompile 'org.assertj:assertj-core:1.7.0'
+  compile 'com.android.support:support-v4:23.1.1'
```

Comment:
You sould change this to :
compile 'com.android.support:support-v4:23.0.1'

To avoid travis error